src/meta/src/rpc/service/ddl_service.rs

```diff
@@ -581,7 +599,9 @@ where
         };
 
         let table_ids_cnt = fragment_graph.table_ids_cnt;
-        let default_parallelism = if self.env.opts.minimal_scheduling {
+        let default_parallelism = if let Some(parallelism) = parallelism {
+            parallelism as usize
+        } else if self.env.opts.minimal_scheduling {
```

Comment:
I guess it's time to remove this option, as it can be achieved by setting the variable, which sounds more flexible.

Reply:
Removed.